docker-compose.yml

```diff
@@ -81,6 +82,20 @@ services:
     env_file: worker.env
     depends_on:
       - memcache
+  worker2:
+    build:
+      context: ./warehouse-workers/workers-v2
+    working_dir: /apps/workers
+    command: node sqsWorker.js
+    volumes:
```

Comment:
Question: Can the `volumes` mapping for `worker2` be simpler like it is for `worker` service or does it need to be specific for local development?

Reply:
Yes, will make it simpler.

Reply:
It can be, but that just makes the code more complex. If I had to just neglect the `warehouse-workers/workers` folder in `worker2` service, then it would have been simpler. But it involves a separate `package.json`, 'Dockerfile` along with that. Going forward, the directory structure can be further simplified, which will make the volumes mounting simpler.